src/localization.ts

```diff
@@ -58,3 +59,30 @@ async function getOriginalLocation(location: Location, projectRoot: string): Pro
         console.error(err);
     }
 }
+
+/**
+ * Translates a static file web URL to local file path.
+ * Assumes that the files are served from ${webUri}/static path
+ */
+function translateToLocalPath(webPath: string, projectRoot: string): string {
+    const [, path] = webPath.split('static/');
+    return `${projectRoot}${projectRoot.endsWith('/') ? '' : '/'}${path}`;
```

Comment:
using the nodejs path api will make this code cleaner, in particular: https://nodejs.org/api/path.html#path_path_join_paths